.github/workflows/release.yml

```diff
@@ -0,0 +1,55 @@
+name: Release
+
+on:
+  push:
+    tags:
+      - "v*"
+
+jobs:
+  build-and-push:
+    name: Build and push container image
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+
+    strategy:
+      matrix:
+        images:
+          - dockerfile: ./build.Dockerfile
+            image_name: ghcr.io/dena/unity-meta-check/unity-meta-check-builder
+          - dockerfile: ./Dockerfile
+            image_name: ghcr.io/dena/unity-meta-check/unity-meta-check
+          - dockerfile: ./.github/images/Dockerfile
+            image_name: ghcr.io/dena/unity-meta-check/unity-meta-check-gh-action
+          - dockerfile: ./.circleci/images/Dockerfile
+            image_name: ghcr.io/dena/unity-meta-check/unity-meta-check-circleci
+
+    steps:
+      - uses: actions/checkout@0ad4b8fadaa221de15dcec353f45205ec38ea70b # v4.1.4
+        with:
+          submodules: recursive
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@68827325e0b33c7199eb31dd4e31fbe9023e06e3 # v3.0.0
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@d70bba72b1f3fd22344832f00baa16ece964efeb # v3.3.0
+
+      - name: Login to ghcr.io
+        uses: docker/login-action@e92390c5fb421da1463c202d546fed0ec5c39f20 # v3.1.0
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Build and push
+        uses: docker/build-push-action@2cdde995de11925a030ce8070c3d77a52ffcf1c0 # v5.3.0
+        with:
+          platforms: linux/amd64,linux/arm64
+          file: ${{ matrix.images.dockerfile }}
+          tags: |
+            ${{ matrix.images.image_name }}:latest
```

Comment:
Up until now, we haven't set prerelease versions (#.#.#-dev# or #.#.#-alpha#) as the `latest`. Is it possible to only push them as the `latest` when they are not prerelease?

Reply:
https://github.com/DeNA/unity-meta-check/pull/39#discussion_r1585899523